.flake8

```diff
@@ -1,19 +1,59 @@
 [flake8]
-# E203 skipped because it annoys black.
-# no pyproject.toml support https://gitlab.com/pycqa/flake8/-/issues/428
-show-source = true
-extend-ignore =
-  E203,
-  F401,
-max-line-length = 100
+
 builtins = _
+
+# Print the total number of errors:
+count = true
+
+# Don't even try to analyze these:
 extend-exclude =
-  # skip tool cache dirs
+  # No need to traverse egg info dir
+  *.egg-info,
+  # tool cache dirs
   *_cache
-  # skip project env vars
+  # project env vars
   .env,
-  # skip adjacent venv
+  # GitHub configs
+  .github,
+  # Cache files of MyPy
+  .mypy_cache,
+  # Cache files of pytest
+  .pytest_cache,
+  # Temp dir of pytest-testmon
+  .tmontmp,
+  # Occasional virtualenv dir
+  .venv
+  # VS Code
+  .vscode,
+  # Temporary build dir
+  build,
+  # This contains sdists and wheels of ansible-navigator that we don't want to check
+  dist,
+  # Metadata of `pip wheel` cmd is autogenerated
+  pip-wheel-metadata,
+  # adjacent venv
   venv
+
+# IMPORTANT: avoid using ignore option, always use extend-ignore instead
+# Completely and unconditionally ignore the following errors:
+extend-ignore =
+  E203,  # skipped because it annoys black
+  F401,  # skipped for unknown reason
```

Comment:
F401 module imported but unused

Since we don't know why it is skipped, and we have no violations, and it _seems_ sane, Should it be enabled, maybe not this PR because I think this is about the format of the file not removals/additions

Reply:
Yep, exactly. I tried to produce minimum influence here but in follow-ups, we should definitely refine what's ignored globally.

Reply:
So this is not forgotten, can you log an issue with a link to this?

Reply:
https://github.com/ansible/ansible-navigator/issues/720